config/environments/test.rb

```diff
@@ -76,6 +76,6 @@
   # Disable PaperTrail by default on tests
   # https://github.com/paper-trail-gem/paper_trail#7-testing
   config.after_initialize do
-    PaperTrail.enabled = ENV['PAPERTRAIL_ENABLED'] || false
+    PaperTrail.enabled = true # ENV['PAPERTRAIL_ENABLED'] || false
```

Comment:
This is only to be sure that other version-related operations still work after this change... this is not going to be merged.